# ROS message containing NMPC inverse objective priorities
# Thomas Stastny | thomas.stastny@mavt.ethz.ch

float32[51] soft_airsp
float32[51] soft_aoa
float32[51] soft_hagl
float32[51] soft_rtd
